### ISSUE TITLE
Namespace compliance report is not visible on rancher 2.7

### ISSUE THREAD

**Issue by kravciak** [rancher/kubewarden-ui#720]
Compliance column showing pass/fail numbers is not visible in namespace view in `rancher 2.7`.
It works fine on `rancher 2.8`.

[Screencast from 2024-05-17 12-22-49.webm](https://github.com/rancher/kubewarden-ui/assets/538604/97370d74-6dc5-455d-a725-307f33b8bafc)

**Comment by jordojordo** [comment]
I don't think this is a bug, AFAIK the PanelColumn extension hook was not added until after 2.7.
